test/Scoop-Decompress.Tests.ps1

```diff
@@ -30,7 +30,7 @@ Describe 'Decompression function' -Tag 'Scoop', 'Decompress' {
     Context '7zip extraction' {
 
         BeforeAll {
-            if ($env:CI) {
+            if ($env:CI_WINDOWS) {
```

Comment:
I don't like the idea of changing `$env:CI` to `$env:CI_WINDOWS`. `$env:CI_WINDOWS` is only available on AppVeyor while `$env:CI` is available on other CIs like GitHub Actions. I'm also doing some changes on tests codes which will introduce the GitHub Actions #4643. It will be reverted in my upcoming commits if it gets changed in this PR.

Reply:
This could be changed back, since the test will only be run on Windows machine ($env:CI_WINDOWS = $true) in `test.ps1`. But for the test in that file, `CI_WINDOWS` is needed unless we force to run the test (using Appveyor or GitHub Actions) on a Windows machine.